### ISSUE TITLE
Has anyone started a firmata library for TinyGo? 

### ISSUE THREAD

**Issue by wade-welles** [hybridgroup/tinygo-zoo#4]
I don't want to re-invent the wheel if it already exists, but I started putting one together last night. And I would like to offer it for use in the primary project, given the popularity of firmata if there is interest and the project is not already being handled by someone else.

**Comment by deadprogram** [comment]
Since you run TinyGo on the microcontroller itself, you would not use Firmata with TinyGo.

Do you mean rewriting Firmata itself using TinyGo?

**Comment by wade-welles** [comment]
Yes, the fimrware component of firmata. It receives midi signals and controls what is happening on the microcontroler.

**Comment by wade-welles** [comment]
https://github.com/firmata/arduino/blob/master/examples/StandardFirmataPlus/StandardFirmataPlus.ino

This example might help make it clearer what Ive been piecing together last few days.
